CHANGELOG.unreleased.md

```diff
@@ -30,6 +32,12 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
+- @xen-orchestra/fs patch
+- vhd-cli patch
+- @xen-orchestra/backups patch
+- @xen-orchestra/backups-cli patch
+- xen-api minor
+- @xen-orchestra/proxy patch
```

Comment:
There are some changes I don't understand, eg `xen-api`.

Reply:
Indeed... It's a residual of a badly managed git conflict.